test/producer-consumer-spec.js

```diff
@@ -14,6 +14,7 @@ let letters = 0;
 /* eslint func-names: "off" */
 /* eslint prefer-arrow-callback: "off" */
 describe('producer/consumer', function () {
+  before(docker.rm);
   before(() => docker.run().then(docker.start));
 
   after(docker.rm);
```

Comment:
it this needed if we have `before(docker.rm)`?

Reply:
If I kill the tests process in the middle (which I do during my development workflow) the next run will fail with - 
```
     ChildProcessError: Command failed: docker run --detach --name=arnavmq-tests -p 5672:5672 rabbitmq:3.8-alpine
docker: Error response from daemon: Conflict. The container name "/arnavmq-tests" is already in use by container "d9945393989ab25489da31522bf865b20d1bc7bac3fa82c8a89e7dec9f416cda". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
 `docker run --detach --name=arnavmq-tests -p 5672:5672 rabbitmq:3.8-alpine` (exited with error code 125)
```

and this because the docker.rm on the `after` wasn't running.

I can remove the `after(docker.rm)` in the project, do you have any idea on how I can make `docker.run()` handle an existing container and not fail?